client/me/purchases/purchase-item/index.jsx

```diff
@@ -540,6 +542,37 @@ class PurchaseItem extends Component {
 		return <SiteIcon site={ site } size={ 36 } />;
 	};
 
+	getProductDisplayName() {
```

Comment:
As far as I can tell, `getProductDisplayName` is just an extension of `getDisplayName` which has conditions for different Jetpack products, correct?

It may make more sense to move this logic into `getDisplayName` under its own AI & Stats condition just to keep things tidy.

Reply:
That's a good point, I'll go ahead and move the logic there 😄